apps/site/lib/site_web/templates/layout/app.html.eex

```diff
@@ -10,6 +10,7 @@
     <% meta_description = assigns[:meta_description] || "Official website of the MBTA -- schedules, maps, and fare information for Greater Boston's public transportation system, including subway, commuter rail, bus routes, and boat lines." %>
     <meta name="description" content="<%= Phoenix.HTML.raw(meta_description) %>">
     <meta name="author" content="Massachusetts Bay Transportation Authority">
+    <meta name="theme-color" content="#071d2e">
```

Comment:
How'd you get this particular hex value? Sorry, we should have been more explicit/linked the ticket to the mocks -- I'm pretty sure the right color is:

```suggestion
    <meta name="theme-color" content="#0b2f4c">
```

*But it works!

Reply:
I was matching off of the footer background color which is what the ss in the ticket appeared to be using, but no problem can swap over to the other